src/main/java/com/iexec/core/workflow/Workflow.java

```diff
@@ -86,12 +85,11 @@ void removeTransition(T from, T to) {
 
     public boolean isValidTransition(T from, T to){
         return possibleTransitions.containsKey(from)
-            ? possibleTransitions.get(from).contains(to)
-            : false;
+                && possibleTransitions.get(from).contains(to);
     }
 
     List<T> toList(T... statuses) {
-        return new ArrayList<T>(Arrays.asList(statuses));
+        return new ArrayList<>(Arrays.asList(statuses));
```

Comment:
```suggestion
        return List.of(statuses);
```
Is this method really necessary ?

Reply:
I don't think so. I'll add a `@Deprecated` annotation. Thanks for pointing it out.

Reply:
Well, I checked the code and we finally need it. The list should be [mutable](https://github.com/iExecBlockchainComputing/iexec-core/blob/91bf3e19ed8f8d0f4248ac7084b20459e390b68f/src/main/java/com/iexec/core/workflow/Workflow.java#L44), so that's probably better to keep this method as is.

Reply:
I missed this, thanks for the counter check